pom.xml

```diff
@@ -228,6 +228,11 @@
       <version>0.10-alpha</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+        <groupId>io.jenkins.plugins</groupId>
+        <artifactId>jaxb</artifactId>
+        <version>2.3.0.1</version>
```

Comment:
Agreed. The docs are indeed misleading.

When I wrote them 1) I had more in mind the end users that still should never have to install this plugin manually and 2) that no plugin would bump that quickly to that baseline, so didn't really care explaining  that indeed at some point plugins should generallly declare explicit dependencies on some detached plugins when they *actually* depend on them.

Thanks a lot for triggering this rather sooner than later Jesse. Better have someone like you hit this first than someone less versed into Jenkins' understandings.